.github/workflows/ci.yml

```
@@ -78,6 +78,7 @@ jobs:
       - name: Install openwisp-monitoring
         run: |
           pip install -U -I -e .
+          pip uninstall -y django
           pip install ${{ matrix.django-version }}
```

```suggestion
          pip install --force-reinstall ${{ matrix.django-version }}
```

I would rather use this one-liner 😄

---

I see that you already tried this.

This is bizarre though. The [logs of failed CI buid](https://github.com/openwisp/openwisp-monitoring/actions/runs/4752709040/jobs/8443391378#step:10:406) shows that correct version of Django was installed 
```
Successfully installed asgiref-3.6.0 django-3.2.18 pytz-2023.3 sqlparse-0.4.4
```

---

@pandafy : Please see CI build status on this change [`f5769bd` (#506)](https://github.com/openwisp/openwisp-monitoring/pull/506/commits/f5769bdcf5015faef209e23d88d551d039c53fd7)

CI build status: https://github.com/openwisp/openwisp-monitoring/actions/runs/4752709040